salt/modules/ansiblegate.py

```diff
@@ -162,9 +166,13 @@ def call(self, module, *args, **kwargs):
             ["echo", "{0}".format(js_args)],
             stdout=subprocess.PIPE, timeout=self.timeout)
         proc_out.run()
+        if six.PY3:
```

Comment:
Reverse the logic here please. If `six.PY2` first. We want to have this code work whenever Py4 comes around.